build/cmake/init.cmake

```diff
@@ -358,6 +359,11 @@ if(wxUSE_GUI)
         set(wxUSE_ENH_METAFILE ON)
     endif()
 
+    if(WXUWP AND wxUSE_METAFILE)
```

Comment:
Could we check for `wxMSW OR WXUWP` above to avoid duplicating this (and potentially other) sections?

Reply:
Yes sure!